UnitTests/tKBO.cpp

```diff
@@ -376,3 +379,210 @@ TEST_FUN(kbo_test23) {
   ASS_EQ(ord.compare(f(alpha), g(beta)), Ordering::Result::INCOMPARABLE)
 }
 
+// isGreater tests
+
+bool isGreaterSymmetric(const KBO& ord, TermList t1, TermList t2) {
+  return ord.isGreater(AppliedTerm(t1),AppliedTerm(t2))
+    && !ord.isGreater(AppliedTerm(t2),AppliedTerm(t1));
```

Comment:
Maybe add 
```
&& ord.compare(t1, t2) == GREATER
&& ord.compare(t2, t1) == LESS
```

Reply:
This is done deliberately to check `isGreater` and that in case of true the opposite relation does not hold.